src/scancode/agent/scancode_template.html

```diff
@@ -0,0 +1,67 @@
+{#
+    SPDX-License-Identifier: GPL-2.0
+    SPDX-FileCopyrightText: 2021 Sarita Singh <saritasingh.0425@gmail.com>  
+
+   This program is free software; you can redistribute it and/or 
+   modify it under the terms of the GNU General Public License 
+   version 2 as published by the Free Software Foundation.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of 
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+   GNU General Public License for more details. 
+
+   You should have received a copy of the GNU General Public License along
+   with this program; if not, write to the Free Software Foundation, Inc.,
+   51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#}
```

Comment:
Can we have a small documentation about this template or something from scancode manuals?

Reply:
Yeah sure! Where should I document these kind of stuff. Will it be okay to add a page in the gsoc doc?

Reply:
Added a small description in the template itself.